get-started/index.md

```diff
@@ -140,3 +140,26 @@ and paste the output into the `Token` input field.
 Now that you have installed and deployed Cryostat and know how to access its
 web client, continue on to [Guides]({% link guides/index.md %}) for
 guides through various common actions and workflows.
+
+## [Uninstalling Cryostat Operator](#uninstalling-cryostat-operator)
+In order to ensure that objects created by the operator and recordings created
+by Cryostat are properly removed, the Cryostat Operator must be installed
+when attempting to delete the Cryostat custom resource, or any Recording
+custom resources.
+
+To completely remove Cryostat and all objects and recordings created by it:
+1. Delete any Recording custom resources.
+    - If Cryostat is no longer deployed, you must redeploy it by creating a
+      Cryostat custom resource.
+    - If the Cryostat Operator has already been uninstalled, please reinstall it.
```

Comment:
"please reinstall it before deleting the Recording custom resource"

Reply:
Done, did the same for the Cryostat CR.